dask_expr/collection.py

```diff
@@ -570,6 +570,11 @@ def dropna(self, how=no_default, subset=None, thresh=no_default):
             expr.DropnaFrame(self.expr, how=how, subset=subset, thresh=thresh)
         )
 
+    def fillna(self, value=None, method=None, limit=None, axis=None):
```

Comment:
This can go to ``Expr``. We try to add things there as long as the signatures are consistent. 

We did not add the axis keyword for now to keep it simple, you can keep it out here too if you don't want to deal with it now.